src/theme/_layout.scss

```
@@ -97,7 +97,8 @@ footer {
 }
 
 // Container definitions for elements
-#page-document > * {
+#page-document > *,
+#page-add > * {
```

```suggestion
#page-document > * {
```

As far as I can see, this isn't needed. There are already CSS rules for block width on the add page. If you found a problem, please show it with a link and screenshot in the issue so that I can understand.